srml/support/procedural/src/storage/transformation.rs

```diff
@@ -500,6 +500,7 @@ fn decl_storage_items(
 			}
 
 			impls.extend(quote! {
+				/// Instance trait implemented by all usable instance of the module.
```

Comment:
`Tag a type as an instance of a module.`

Maybe something like this?

Reply:
yes, I can go for that, but I thought tag was more for empty trait, Instance trait still contains some const for each instance storage keys.
```suggestion
				/// Tag a type as an instance of a module.
				/// 
				/// Defines storage prefixes that must be unique.
```